### PR TITLE
Add WakaTime extension

### PR DESCRIPTION
**Pull request by gandarez**
This PR adds WakaTime extension to the list.

@alanrenmsft @llali

### CODE REVIEW

File: extensionsGallery.json [hunk 1]
```diff
@@ -3297,6 +3297,63 @@
 					],
 					"statistics": [],
 					"flags": ""
+				},
+				{
+					"extensionId": "70",
```
Comment (alanrenmsft) [inline extensionsGallery.json:3302]: please use the same id 71 here.

File: extensionsGallery-insider.json [hunk 2]
```diff
@@ -3415,15 +3415,68 @@
 					],
 					"statistics": [],
 					"flags": "preview"
-				}
+				},
+				{
+					"extensionId": "71",
+					"extensionName": "vscode-wakatime",
+					"displayName": "WakaTime",
+					"shortDescription": "Automatic time tracking and quantified-self metrics generated from your SQLOps usage.",
```
Comment (alanrenmsft) [inline extensionsGallery-insider.json:3423]: from your SQLOps usage -> from your Azure Data Studio usage?

**Comment by gandarez** [comment]
@alanrenmsft fixes applied.

**Comment by alanrenmsft** [comment]
@gandarez I had to make some additional changes after the changes are deployed for insider feed: https://github.com/microsoft/azuredatastudio/commit/c7a584b16402d0b760e47155ee9485ec7db56fdb

now this extension is available for both stable and insider builds. 🎉

**Comment by gandarez** [comment]
@alanrenmsft thanks for merging it. Just a question isn't there a way to automatically install the extension (ad vscode does) instead of asking to open an external website?

**Comment by alanrenmsft** [comment]
@gandarez unfortunately, we don't have a marketplace like vscode does, due to security reasons, we have to ask the users to download the 3rd party extensions. we are still working with the vscode team and hopefully we can onboard their marketplace.

**Pull request merged**